pkgs/by-name/fl/flashgbx/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, copyDesktopItems
+, makeDesktopItem
+, qt6
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "flashgbx";
+  version = "3.37";
+
+  src = fetchFromGitHub {
+    repo = "FlashGBX";
+    owner = "lesserkuma";
+    rev = version;
+    hash = "sha256-3527QmSSpGotFHKTg0yb6MgHKSze+9BECQWqZM3qKsw=";
+  };
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "flashgbx";
+      desktopName = "FlashGBX UI";
+      icon = "flashgbx";
+      exec = meta.mainProgram;
+      comment = "UI for reading and writing Game Boy and Game Boy Advance cartridges";
+      categories = [ "Utility" ];
+    })
+  ];
+
+  postInstall =
+  ''
+      install -D FlashGBX/res/icon.png $out/share/icons/hicolor/256x256/apps/flashgbx.png
+  '';
+
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
https://github.com/NixOS/nixpkgs/issues/253154#issuecomment-1712673789

---

Okay, thanks for this and the reference. I have addressed this in the latest push.